### ISSUE TITLE
Card Tsar is sometimes resigned even though they can't act

### ISSUE THREAD

**Issue by grifferz** [grifferz/pah-irc#64]
- `21:24:49 < AgainstHumanity>` **4** people recently played! We're currently waiting on plays from **2** people: **prando** and **paul**.
- `09:01:52 < AgainstHumanity>` I'm forcibly resigning **tommy** from the game due to idleness. Idle since **2015-02-19T22:09:48**.
- `09:01:54 < AgainstHumanity>` The new Card Tsar is **dave2**. Time for the next Black Card:
- `09:02:17 < AgainstHumanity>` → **The school trip was completely ruined by _____.**

Here, Tommy could not take any action (because the game was waiting on Prando and Paul), but waS resigned anyway. One of Prando or paul should have been resigned.
